reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -661,13 +661,8 @@ private fun ResolvedLicenseLocation.permalink(id: Identifier): String? {
         }
 
         vcsInfo != null && vcsInfo != VcsInfo.EMPTY -> {
-            val path = listOfNotNull(
-                vcsInfo.path.takeIf { it.isNotEmpty() },
-                location.path
-            ).joinToString("/")
-
             VcsHost.toPermalink(
-                vcsInfo.copy(path = path),
+                vcsInfo.copy(path = location.path),
```

Comment:
Does this need a Git Repo special treatment?

Reply:
I don't think so, because there is no `VcsHost` implementation that supports repo, and there likely never will be. And if there was one, it would have to implement `toPermalink` correctly so that this works.